VocaDb.ResXFileCodeGenerator.sln.DotSettings

```diff
@@ -0,0 +1,196 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Personally, I haven't used ReSharper, so I'm wondering what the purpose of this file is. Is this different from `.editorconfig`?

Reply:
No. Pretty much the same. Although i did autogen the current .editorconfig from resharper, so there shouldn't be any settings that differ. I do highly recommend resharper, it has a ROI measured in days.